### ISSUE TITLE
SegmentationFault with any use of fetch on M1 Pro

### ISSUE THREAD

**Issue by tascodes** [oven-sh/bun#661]
### Version

0.1.3

### Platform

Darwin Tas-MacBook-Pro.local 21.5.0 Darwin Kernel Version 21.5.0: Tue Apr 26 21:08:37 PDT 2022; root:xnu-8020.121.3~4/RELEASE_ARM64_T6000 arm64

### What steps will reproduce the bug?

**src/index.ts**
```typescript
async function checkStock() {
  return await fetch("https://example.com");
}

checkStock().then((res) => {});
```

Run:
```
bun run src/index.ts
```

### How often does it reproduce? Is there a required condition?

Reproduces 8 times out of 10, approximately. No clear specific condition.

### What is the expected behavior?

The program should execute and exit with status 0. 

### What do you see instead?

```
SegmentationFault at 26


–––– bun meta ––––
Bun v0.1.3 macOS Silicon 21.5.0
RunCommand:
Elapsed: 285ms | User: 203ms | Sys: 71ms
RSS: 0.20GB | Peak: 0.20GB | Commit: 71.32MB | Faults: 1
–––– bun meta ––––
```

**Comment by hisamafahri** [comment]
Probably duplicate/related to #337 #614 and #607.

**Comment by FinnRG** [comment]
This was resolved with version ``0.1.4``. Please upgrade to the latest version.
